doc/configuration/EDRConfiguration/EDR.md

```diff
@@ -8,17 +8,32 @@ The OGC API EDR service can be configured in a dataset configuration by adding t
 
 ```xml
     <OgcApiEdr>
-        <EdrCollection name="harmonie">
-            <EdrParameter name="air_temperature__at_2m" unit="Celsius"/>
+        <EdrCollection name="harmonie" time_interval="R48/{reference_time}+1/PT60M" vertical_name="height" >
+            <EdrParameter name="air_temperature__at_height" unit="Celsius" standard_name="air_temperature"  parameter_label="Temperature of air" observed_property_label="Air temperature"/>
+        </EdrCollection>
+        <EdrCollection name="....">
+          <EdrParameter name="..."/>
         </EdrCollection>
     </OgcApiEdr>
 ```
 
 EdrCollection:
 - name: The name of the EDR collection
-- vertical_dimension?: For example pressure_level
+- vertical_name?: For example pressure_level, the name of the vertical dimension in the WMS layer configuration
```

Comment:
Did you check `Configure_EDR_service.md` as well?

Reply:
Yes, added a few lines.